### PR TITLE
Split Flask Dapr from Dapr

### PR DESCRIPTION
**Pull request by chinzhiweiblank**
# Description

I have added the following to the `flask_dapr` folder.

1. setup.py
2. setup.cfg
3. version.py
4. LICENSE (Autogenerated from the main Dapr LICENSE)

I have also added the ignoring of the `flask_dapr` folder in the main Dapr package.
I have also added the building of the flask_dapr package in the `build.yaml`.

## Issue reference

We strive to have all PR being opened based on an issue, where the problem or feature have been discussed prior to implementation.

Please reference the issue this PR will close: #53 

## Checklist

Please make sure you've completed the relevant tasks for this PR, out of the following list:

* [X] Code compiles correctly
* [ ] Created/updated tests
* [ ] Extended the documentation

### CODE REVIEW

File: .github/workflows/build.yaml [hunk 1]
```diff
@@ -36,11 +36,20 @@ jobs:
...
       env:
         TWINE_USERNAME: ${{ secrets.PYPI_UPLOAD_USER }}
         TWINE_PASSWORD: ${{ secrets.PYPI_UPLOAD_PASS }}
       run: |
         python setup.py sdist bdist_wheel
         twine upload dist/*
+    - name: Build and publish Dapr Flask Extension
+      if: github.event_name != 'pull_request'
+      env:
+        TWINE_USERNAME: ${{ secrets.PYPI_UPLOAD_USER }}
```
Comment (youngbupark) [inline .github/workflows/build.yaml:50]: let's move these environment variables under `build:` env section.

File: setup.py [hunk 2]
```diff
@@ -51,4 +51,5 @@ def is_release():
     version=version,
     description=description,
     long_description=long_description,
+    packages=find_packages(exclude=("flask-dapr",))
```
Comment (youngbupark) [inline setup.py:54]: can you specify this option inside setup.cfg ? I would like to minimize the setting inside setup.py.

File: flask_dapr/setup.cfg [hunk 3]
```diff
@@ -0,0 +1,38 @@
+[metadata]
+url = https://dapr.io/
+author = Dapr
+author_email = daprweb@microsoft.com
+license = MIT
+license_file = ../LICENSE
```
Comment (youngbupark) [inline flask_dapr/setup.cfg:6]: you might want to refer to LICENSE in flask_dapr.

**Comment by chinzhiweiblank** [comment]
@youngbupark Made the changes.

**Comment by youngbupark** [comment]
> @youngbupark Made the changes. Would you also like to remove Flask from Dapr's `setup.cfg`?

Yeah make sense.

**Pull request merged**